.gitignore

```diff
@@ -81,4 +81,8 @@ tags
 ### VisualStudioCode ###
 .vscode/*
 .history
+
+### Test logs ###
```

Comment:
I have a personal vendeta against comments that add zero information.

Reply:
thought it is a convention here

Reply:
maybe it's just me. stating the obvious is tiring.